test/new-e2e/tests/agent-metric-logs/customkind/kind.go

```diff
@@ -0,0 +1,210 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package customkind spins up the same pulumi environment as the awskubernetes package but includes
+// a logger container as well
+package customkind
```

Comment:
Would it make sense to call this something more descriptive? Like "awswithlogger" or just "kindwithlogger".